Documentation/doc/Documentation/windows.txt

```diff
@@ -3,7 +3,7 @@
 \cgalAutoToc
 
 \cgal \cgalReleaseNumber is supported for the following \ms Visual `C++` compilers:
-14.0, 15.9, 16.0 (\visualstudio 2015, 2017, and 2019).
+14.0, 15.9, 16.0 (\visualstudio 2015, 2017, 2019, and 2022).
 
```

Comment:
Shouldn't the version 19.0 be mentioned as well (now 3 versions and 4 years mentioned).